duckduckgo/iOS
# Update GPC copy

Task/Issue URL: https://app.asana.com/0/inbox/856498667309057/1199892618151770/1200330230492734
Tech Design URL:
CC:

**Description**:
Update GPC copy 

**Steps to test this PR**:

- Validate if GPC copy matches specification.
- Ensure GPC info link is correct.
- Check if translations work as expected.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
